lib/gui/app.js

```diff
@@ -219,3 +220,47 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
   };
 
 });
+
+app.controller('ApplicationController', function($state,
+                                          SupportedFormatsModel,
+                                          OSDialogService,
+                                          AnalyticsService,
+                                          SelectionStateModel) {
+
+  this.formats = SupportedFormatsModel;
+
+  /**
+   * @summary Select image
+   * @function
+   * @public
+   *
+   * @param {Object} image - image
+   *
+   * @example
+   * OSDialogService.selectImage()
```

Comment:
This example is a bit confusing. We show a dialog to then "drop" the file. Maybe its clearer to simply pass a mock image object to the function?

```javascript
ApplicationController.dropImage({
   path: 'path/to/image.img',
   size: 999999
});
```